site/src/api/api.ts

```diff
@@ -601,20 +601,20 @@ class ApiMethods {
   };
 
   patchOrganizationRole = async (
-    organizationId: string,
+    organization: string,
```

Comment:
can you copy-paste the doc comment a lot of these other org functions have? it's a minor thing, but I like the easily accessible reassurance that it can be the id or the name 😄